### ISSUE TITLE
ClipEdges is not working anymore

### ISSUE THREAD

**Issue by maidi29** [ThatOpen/engine_components#452]
### Describe the bug 📝

When having EdgesPlane as clipper.Type and styles for it it is not working anymore to display these styles.

It is also visible in the example tutorial where the cubes earlier had colored borders but right now they have not
<img width="889" alt="image" src="https://github.com/user-attachments/assets/5dd801c9-e17e-4e62-8d87-c85990a99a7a">

### Reproduction ▶️

https://docs.thatopen.com/Tutorials/Components/Front/ClipEdges

### Steps to reproduce 🔢

Add a clipping plane to the boxes

### System Info 💻

```shell
"@thatopen/components": "2.1.14",
"@thatopen/fragments": "2.1.4",
"@thatopen/components-front": "2.1.8",
```

### Used Package Manager 📦

npm

### Validations ✅

- [X] Read the [docs](https://docs.thatopen.com/intro).
- [X] Check that there isn't [already an issue](https://github.com/ThatOpen/engine_components/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a repository issue and not a framework-specific issue. For example, if it's a THREE.js related bug, it should likely be reported to [mrdoob/threejs](https://github.com/mrdoob/three.js) instead.
- [X] Check that this is a concrete bug. For Q&A join our [Community](https://people.thatopen.com/).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by agviegas** [comment]
Fixed from `@thatopen/components-front@2.1.10`!

**Comment by maidi29** [comment]
Works perfectly, thanks!
